content/en/user-guide/aws/kinesis/index.md

```diff
@@ -39,12 +39,19 @@ $ awslocal kinesis put-record --stream-name samplestream --data '{"symbol":"TEST
 
 ## Configuration
 
-
 Regardless of which provider you're working with, Kinesis can be further configured with the following environment variables:
 
 | Variable | Description |
 | -------- | ----------- |
 | `KINESIS_ERROR_PROBABILITY` | Decimal value between `0.0` (default) and `1.0`. Typically, it is difficult to know beforehand whether your application can handle the throughput and whether it can deal with backpressure. By setting this environment variable the application will randomly inject `ProvisionedThroughputException`. While this won't tell you whether your application can handle sufficient throughput, it does help you to test whether your application can handle exceptions gracefully. |
 | `KINESIS_SHARD_LIMIT` | Integer value (default: `100`) or `Infinity` (to disable). This variable can help you to test whether your application adheres to the allocated shard limit. This behavior can only be disabled by explicitly setting the environment variable as `KINESIS_SHARD_LIMIT=Infinity` |
 | `KINESIS_LATENCY` | Integer value of milliseconds (default: `500`) or `0` (to disable). Especially important for testing latency-critical applications. Since latency cannot be tested with the local Kinesis service, you can use this variable to introduce artificial latency into your AWS calls. This behavior can only be disabled by explicitly setting the environment variable as `KINESIS_LATENCY=0`. |
-| `KINESIS_INITIALIZE_STREAMS` | A comma-delimited string of stream names, its corresponding shard count and an optional region to initialize during startup. If the region is not provided, the default region is used. For instance, `KINESIS_INITIALIZE_STREAMS=my-first-stream:1,my-other-stream:2:us-west-2,my-last-stream:1` **Important**: Only supported by `kinesis-mock` |
+| `KINESIS_INITIALIZE_STREAMS` | **Deprecated** A comma-delimited string of stream names, its corresponding shard count and an optional region to initialize during startup. If the region is not provided, the default region is used. For instance, `KINESIS_INITIALIZE_STREAMS=my-first-stream:1,my-other-stream:2:us-west-2,my-last-stream:1`. In multi-account setups, the streams will be created for all accounts. |
+
+## Limitations
+
+In multi-account setups, each AWS account launches a separate instance of Kinesis Mock.
+This is very resource intensive when a large number of AWS accounts are used.
+
+This will be addressed in a future update.
+Also see <https://github.com/etspaceman/kinesis-mock/issues/377>.
```

Comment:
Instead of putting a direct link here, can we rephrase it as `Follow up with the following [GitHub Issue](https://github.com/etspaceman/kinesis-mock/issues/377) to keep track of the development`.